test/format.test.ts

```diff
@@ -180,3 +180,27 @@ describe("xlsx-wasm test", () => {
     expect(actual).matchXlsx(expected);
   });
 });
+
+describe("xlsx-wasm test", () => {
+test("issue6", async () => {
+    // Arrange
+    const workbook = new Workbook();
+
+    // Act
+    const worksheet = workbook.addWorksheet();
+    const format = new Format();
+    format
+      .setFontName('Meiryo UI')
+      .setFontSize(16)
+      .setAlign(FormatAlign.Center)
+      .setBorder(FormatBorder.Thin);
+
+    worksheet.writeWithFormat(0, 0, 'foo', format);
+    worksheet.writeWithFormat(1, 1, 'bar', format);
+
+    // Assert
+    const actual = await readXlsx(workbook.saveToBufferSync());
+    const expected = await readXlsxFile("./expected/issue6.xlsx");
```

Comment:
Would you mind using a clearer name so that others can easily understand what is being tested?

Reply:
fixed